packages/vite/src/node/logger.ts

```diff
@@ -158,19 +159,31 @@ export function createLogger(
   return logger
 }
 
-export function printServerUrls(
+export function printServerInfo(
```

Comment:
IMO, we don't have to have a dedicated shortcut to show loaded env files and it's fine to print them with other info on server start and on demand. 

What do you think?

Reply:
we could also print them indeed, maybe a shortcut is overengineered